sdks/rust/src/alpha.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use grpcio;
+
+use errors::*;
+use grpc::alpha;
+use grpc::alpha_grpc;
+
+/// Alpha is an instance of the Agones Alpha SDK
+pub struct Alpha {
+    client: Arc<alpha_grpc::SdkClient>,
+}
+
+impl Alpha {
+    /// new creates a new instance of the Alpha SDK
+    pub fn new(ch: grpcio::Channel) -> Alpha {
+        let cli = alpha_grpc::SdkClient::new(ch);
+        Alpha {
+            client: Arc::new(cli),
+        }
+    }
+
+    /// This returns the last player capacity that was set through the SDK.
+    /// If the player capacity is set from outside the SDK, use sdk.get_gameserver() instead.
+    pub fn get_player_capacity(&self) -> Result<i64> {
+        let req = alpha::Empty::new();
+        let count = self.client.get_player_capacity(&req).map(|c| c.count)?;
+        Ok(count)
+    }
+
+    /// This changes the player capacity to a new value.
+    pub fn set_player_capacity(&self, capacity: i64) -> Result<()> {
+        let mut c = alpha::Count::new();
+        c.set_count(capacity);
+        let res = self.client.set_player_capacity(&c).map(|_| ())?;
+        Ok(res)
```

Comment:
```suggestion
        self.client.set_player_capacity(&c)
```

Could this be reduced to a single line? It _looks_ like the types line up, but figured I would ask.

Same for similar things below.

Reply:
self.client.set_player_capacity returns grpcio::Result<super::alpha::Empty> in the code generated by protoc, but SDK returns Result<()>.
Therefore it cannot be changed as such.
https://github.com/googleforgames/agones/blob/8b4c100070cfacb31395015acdbd2b7d19262567/sdks/rust/src/grpc/alpha_grpc.rs#L132

I could change it to something like below, but I feel the brevity doesn't change much...
```rs
        self.client
            .set_player_capacity(&c)
            .map(|_| ())
            .map_err(|e| Error::from(e))
```

Reply:
Got it. That makes total sense.